scripts/upgrade

```diff
@@ -220,7 +221,7 @@ fi
 #=================================================
 ynh_script_progression --message="Starting a systemd service..." --time --weight=1
 
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line-match="[INFO] Available at http://:::9000/ in private mode"
```

Comment:
Same as the other command

(btw, it feels weird that this is a start action instead of a reload or restart, considering that this is the upgrade script and the service should already be started ? But maybe I'm missing something)

```suggestion
ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="[INFO] Available at http://:::9000/ in private mode"
```